Library/Homebrew/cmd/commands.rb

```diff
@@ -70,6 +70,6 @@ def find_internal_commands(directory)
       end
     end
 
-    private :internal_commands, :internal_developer_commands, :external_commands, :find_internal_commands
+    # private :internal_commands, :internal_developer_commands, :external_commands, :find_internal_commands
```

Comment:
You can call private methods with e.g. `myobj.send(:my_private_method, ...args...)`.